ui/src/ducks/app/monitoring.js

```diff
@@ -39,23 +42,40 @@ export const setClusterStatusAction = payload => {
   return { type: SET_CLUSTER_STATUS, payload };
 };
 
-export function* fetchAlerts() {
-  const api = yield select(state => state.config.api);
-  const result = yield call(getAlerts, api.url_prometheus);
-  if (!result.error) {
-    yield put(setAlertsAction(result.data.data.alerts));
-  }
-}
-
 export function* fetchClusterStatus() {
-  const api = yield select(state => state.config.api);
-  const result = yield call(getClusterStatus, api.url_prometheus);
-  if (!result.error) {
-    yield put(setClusterStatusAction(result.data.data.result));
+  const clusterHealth = {
+    status: '',
+    label: ''
+  };
+  const resultAlerts = yield call(getAlerts); // Check if Prometheus API is available
```

Comment:
Why should we rely on the alert list to check if Prom API is available? I think there's a simpler `/health` endpoint for this purpose, which you could poll every `N` seconds and store the result in Redux state.

Otherwise, a good pattern is to try to get a result, and react accordingly if it fails:

> **It's easier to ask forgiveness than it is to get permission.**
> __ Grace Hopper

In any case, I'm not sure merging the two sagas is a good idea.

Reply:
We do not have any endpoint like /health :(